packages/react-core/src/layouts/Level/Level.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/patternfly-next/layouts/Level/styles.css';
+
+const propTypes = {
+  hasGutter: PropTypes.bool,
```

Comment:
gutters will have sizes in coming releases.  Should follow how Gallery is setup: https://github.com/patternfly/patternfly-react/pull/436/files#diff-7feb448891f41d821b87a088742af454R15 This will be moved to a shared area with a Grid PR that is going to follow Gallery.

Reply:
Oh, that's cool, I'll change it to use different gutter values.